libs/ui-lib/lib/common/components/clusterWizard/clusterDetailsValidation.ts

```diff
@@ -123,28 +120,6 @@ export const getClusterDetailsValidationSchema = ({
         ? baseDomainValidationSchema.required('Required')
         : dnsNameValidationSchema.required('Required'),
       pullSecret: pullSecretValidationSchema.required('Required.'),
-      SNODisclaimer: Yup.boolean().when(['controlPlaneCount', 'openshiftVersion'], {
```

Comment:
Also realised that there's no SNO disclaimer checkbox in CIM, so no need for keeping this code.